app/models/event_invoice.py

```diff
@@ -25,7 +12,7 @@ class EventInvoice(SoftDeletionModel):
     __tablename__ = 'event_invoices'
 
     id = db.Column(db.Integer, primary_key=True)
-    identifier = db.Column(db.String, unique=True, default=get_new_identifier)
+    identifier = db.Column(db.String, unique=True, default=get_new_identifier())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'model' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5550207)